core/service/src/lib.rs

```diff
@@ -23,13 +25,14 @@ pub mod util {
 pub mod cryptography {
     #[cfg(feature = "non-wasm")]
     pub mod attestation;
-    pub mod backup_pke;
     pub mod decompression;
     pub mod error;
     pub mod hybrid_ml_kem;
+    pub mod internal_crypto_types;
+    pub mod signatures;
     // Allow our deprecated modules for now as we need to be backwards compatible
     #[allow(deprecated)]
-    pub mod internal_crypto_types;
+    pub mod encryption;
     #[allow(deprecated)]
     pub mod signcryption;
```

Comment:
and here

Reply:
Yes. Otherwise clippy becomes very unhappy